includes/namespace.php

```diff
@@ -103,6 +103,14 @@ function activate( $network_wide = false ) {
 		);
 	}
 
+	if ( version_compare( get_bloginfo( 'version' ), WEBSTORIES_MINIMUM_WP_VERSION, '<' ) ) {
```

Comment:
One thing:

I am currently on WordPress 5.5.

If I change `WEBSTORIES_MINIMUM_WP_VERSION` to `5.5.0`, this will error because 5.5 < 5.5.0.

How should we handle that? Just document this properly that this needs to be either `5.3` if meaning `5.3.0`, and `5.3.1` if also requiring a certain minor version?

Reply:
As I remember, you don't need the `.0` at the end. I only added it as the line above had it. So I have removed it from both places. https://github.com/google/web-stories-wp/pull/3907/commits/bf55aae88096e043ac27c7cfc03067a7d6b7f45a